my_tests/TransientFEsTests/BoundaryHeatEquationTests.jl

```diff
@@ -0,0 +1,97 @@
+using Pkg
```

Comment:
The whole my_tests folder should not be pushed to the main gridap project

Reply:
started clean branch of master fork. my_tests folder is removed